opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -268,6 +269,22 @@ namespace cpgrid
             // removeUnusedNodes(output);
         }
 
+        {
+            std::vector<int> new_actnum(global_nc, 0);
+            for (int i = 0; i < output.number_of_cells; ++i) {
+                new_actnum[output.local_cell_index[i]] = 1;
+            }
+            const auto& ecl_grid = ecl_state.getInputGrid();
+            const auto& fp = ecl_state.fieldProps();
+            aquifer.mutableNumericalAquifers().addAquiferConnections(deck, ecl_grid, new_actnum);
+            const auto& aquifer_nnc = aquifer.numericalAquifers().aquiferConnectionNNCs(ecl_grid, fp);
+            // We need to update the nnc in the ecl_state
+            ecl_state.appendInputNNC(aquifer_nnc);
```

Comment:
Only process zero can reach here. It means other processes might not have this updated NNC information.

Reply:
It is fixed in the ebos side.